app/src/js/components/Collections/overview.js

```diff
@@ -86,20 +98,13 @@ const CollectionOverview = ({
     historyPushWithQueryParams(collectionHrefFromId(newCollectionId));
   }
 
-  function generateBulkActions() {
-    return [
-      reingestAction(granules),
-      {
-        Component: (
-          <Bulk
-            element="a"
-            className="button button__bulkgranules button--green button--small form-group__element link--no-underline"
-            confirmAction={true}
-          />
-        ),
-      },
-    ];
-  }
+  useEffect(() => {
+    dispatch(listWorkflows());
+  }, [dispatch]);
+
+  useEffect(() => {
+    setWorkflow(workflowOptions[0]);
+  }, [workflowOptions]);
```

Comment:
I think we need to do `JSON.stringify(workflowOptions)` here so it updates properly